### PR TITLE
issue 961, add spider for jason's deli

### LINKED ISSUE #961: Jason's Deli
https://www.jasonsdeli.com/restaurants

### PR DESCRIPTION
**Pull request by gmavian**
closes #961

### CODE REVIEW

File: locations/spiders/jasonsdeli.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            'city': response.xpath('//div[@class="address"]/text()').extract()[-1].split(',')[0],
+            'state': response.xpath('//div[@class="address"]/text()').extract()[-1].split(', ')[1].split(' ')[-2],
+            'postcode': response.xpath('//div[@class="address"]/text()').extract()[-1].split(', ')[1].split(' ')[-1],
+            'ref': ref,
+            'website': response.url,
+            'phone': response.xpath('//a[@class="cnphone"]/text()').extract_first()
+        }
+
+        hours = self.parse_hours(response.xpath('//div[@class="loc-hours"]/p/text()').extract())
+        print(hours)
```
Comment (iandees) [inline locations/spiders/jasonsdeli.py]: Please remove the print.

Comment (gmavian) [reply locations/spiders/jasonsdeli.py]: i think i forgot to convert to 24h format too.. will retest over the weekend

**Comment by gmavian** [comment]
requested changes made and tested. let me know if i goofed with the other commits that got in to this branch :/

**Comment by gmavian** [comment]
gonna close and resubmit this. somehow made a mess of my local branch while getting set up on my laptop

**Pull request closed without merging**